modules/db_instance/main.tf

```diff
@@ -2,38 +2,11 @@ locals {
   is_mssql = element(split("-", var.engine), 0) == "sqlserver"
 }
 
-data "aws_iam_policy_document" "enhanced_monitoring" {
-  statement {
-    actions = [
-      "sts:AssumeRole",
-    ]
+resource "random_id" "snapshot_identifier" {
+  count = var.create && !var.skip_final_snapshot ? 1 : 0
 
-    principals {
-      type        = "Service"
-      identifiers = ["monitoring.rds.amazonaws.com"]
-    }
-  }
-}
-
-resource "aws_iam_role" "enhanced_monitoring" {
-  count = var.create_monitoring_role ? 1 : 0
-
-  name               = var.monitoring_role_name
-  assume_role_policy = data.aws_iam_policy_document.enhanced_monitoring.json
-
-  tags = merge(
-    {
-      "Name" = format("%s", var.monitoring_role_name)
-    },
-    var.tags,
-  )
-}
-
-resource "aws_iam_role_policy_attachment" "enhanced_monitoring" {
-  count = var.create_monitoring_role ? 1 : 0
-
-  role       = aws_iam_role.enhanced_monitoring[0].name
-  policy_arn = "arn:${var.iam_partition}:iam::aws:policy/service-role/AmazonRDSEnhancedMonitoringRole"
+  keepers     = { id = var.identifier }
```

Comment:
Please split this into several lines.

Reply:
fixed in https://github.com/terraform-aws-modules/terraform-aws-rds/pull/314/commits/9a6243be1ef5e37910f3d343a84d790f096733bd